auth0/auth0_client_test.go

```diff
@@ -0,0 +1,88 @@
+package auth0
+
+import (
+	"github.com/google/uuid"
+	"os"
+	"sync"
+	"testing"
+	"time"
+)
+
+// 1. Create Token
+// 2. Create new user
+// 3. Simulate throttled load to GetUserById
+// 4. Clean up the created user
+func TestAccGetUserByIdIsNotThrottled(t *testing.T) {
```

Comment:
`TestAccGetUserByIdIsNotThrottled` change to `TestAccGetUserByIdIsNotRateLimited`